frontend/src/components/permissionsBlock.js

```diff
@@ -0,0 +1,44 @@
+import React, { useContext } from 'react';
+import { FormattedMessage } from 'react-intl';
+import messages from './projectEdit/messages.js'
+import { StateContext, styleClasses } from '../views/projectEdit'
+
+export const PermissionsBlock = ({ permissions, type }: Object) => {
+    const { projectInfo, setProjectInfo } = useContext(StateContext);
```

Comment:
Another small detail to improve. You're mixing 4 spaces indentation with 2 spaces.